### PR TITLE
DM-34805  Add support for a plan as well as a plan report 

### PR DESCRIPTION
**Pull request by womullan**
This will be needed with a new version of Docsteady to give Holger a PLAN and a report .. i got some scons errors in a different part of the template ..

### CODE REVIEW

File: project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}plan.tex [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+\maketitle
+
+\section{Introduction}
+Use the GitHub Actions workflow ``docgen from Jira'' to generate this document's content from Jira.
+
+\subsection{References}
+\label{sect:references}
+\renewcommand{\refname}{}
```
Comment (jonathansick) [inline project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}plan.tex:39]: Why is this here? It is left-over from experimentation, or something that should go in the `lsstdoc` class instead?

File: project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}plan.tex [hunk 2]
```diff
@@ -0,0 +1,44 @@
+\documentclass[DM,lsstdraft,toc]{lsstdoc}
```
Comment (jonathansick) [inline project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}plan.tex:1]: Double check the default subsystem be `SE`?

File: project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/.github/workflows/ci.yaml [hunk 3]
```diff
@@ -38,4 +38,4 @@ jobs:
           LTD_PASSWORD: {% raw %}${{ secrets.LTD_PASSWORD }}{% endraw %}
           LTD_USERNAME: {% raw %}${{ secrets.LTD_USERNAME }}{% endraw %}
         run: |
-          lander --upload --pdf {{ cookiecutter.series }}-{{ cookiecutter.serial_number }}.pdf --lsstdoc {{ cookiecutter.series }}-{{ cookiecutter.serial_number }}.tex --ltd-product {{ cookiecutter.series.lower() }}-{{ cookiecutter.serial_number }}
+          lander --upload --pdf {{ cookiecutter.series }}-{{ cookiecutter.serial_number }}.pdf --lsstdoc {{ cookiecutter.series }}-{{ cookiecutter.serial_number }}.tex --ltd-product {{ cookiecutter.series.lower() }}-{{ cookiecutter.serial_number }} --extra-download  {{ cookiecutter.series.lower() }}-{{ cookiecutter.serial_number }}plan.pdf
```
Comment (jonathansick) [inline project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/.github/workflows/ci.yaml:41]:
```suggestion
          lander --upload --pdf {{ cookiecutter.series }}-{{ cookiecutter.serial_number }}.pdf --lsstdoc {{ cookiecutter.series }}-{{ cookiecutter.serial_number }}.tex --ltd-product {{ cookiecutter.series.lower() }}-{{ cookiecutter.serial_number }} --extra-download  {{ cookiecutter.series }}-{{ cookiecutter.serial_number }}plan.pdf
```

Ensure that the plan's filename matches the templated file.

**Review by jonathansick — COMMENTED** [review]
After adjusting based on the review, please rebase against the latest `main` branch.

Besides the filename issue for `--extra-download`, I think we could make  the plan's filename slightly more aesthetic by adding a hyphen before plan e.g. `EXAMPLE-123-plan.pdf`.

> i got some scons errors in a different part of the template ..

Please elucidate? GitHub Actions does not show "scons errors".

File: project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}plan.tex (hunk 1 above)
Comment (womullan) [reply project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}plan.tex:39]: will check that - its a copy of the other doc in there.. scons was that i neede python 3.10 ok now

File: project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}plan.tex (hunk 2 above)
Comment (womullan) [reply project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}plan.tex:1]: i did not change it

File: project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}plan.tex (hunk 1 above)
Comment (womullan) [reply project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}plan.tex:39]: \renewcommand{\refname}{} is needed in this doc since the bib is in a subseciton if we have to supress the automatic unlabled section

**Comment by womullan** [comment]
rebased
